CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 3.0.5.dev0
```

Comment:
Since this is a behavior change, I think we should play safe here and merge this into `features`. Could you please change the target branch of the PR?